Realm/RLMCollection.mm

```diff
@@ -426,11 +428,19 @@ - (void)invalidate {
     }
     bool skipFirst = std::is_same_v<RLMCollection, RLMResults>;
     auto token = [[RLMCancellationToken alloc] init];
+    
+    realm::KeyPathArray keyPathArray;
+    for (NSString *keyPath in keyPaths) {
+        // ???: Is it possible to hit this when a collection doesn't have an RLMClassinfo?
+        RLMClassInfo *info = collection.objectInfo;
+        RLMKeyPath rlmKeyPath = RLMKeyPathFromString(realm.schema, info->rlmObjectSchema, info, keyPath);
+        keyPathArray.push_back(rlmKeyPath);
```

Comment:
This is copying the keypath, and should either eliminate the local variable or move from it.

Reply:
Resolved in [`8670ce2`](https://github.com/realm/realm-cocoa/pull/7249/commits/8670ce267eb70c71ef44e1000950aac06ea0a599), If I understand you right.